Dockerfile.build

```diff
@@ -9,7 +9,7 @@ CMD ["/usr/bin/bash"]
 ENV LANG=en_US.utf8
 ENV USER_NAME=forge
 #ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 8.9.4
+ENV NODE_VERSION 10.3.0
```

Comment:
You need to update the list of GPG keys below too

Reply:
ok ;)